doc/api/events.md

```diff
@@ -307,6 +307,45 @@ to each.
 
 Returns `true` if the event had listeners, `false` otherwise.
 
+
+```js
+const EventEmitter = require('events');
+const myEmitter = new EventEmitter();
+
+// First listener
+myEmitter.on('event', function firstListener() {
+  console.log('Helloooo! first listener');
+});
+// Second listener
+myEmitter.on('event', function secondListener(arg1, arg2) {
+  console.log(`event with parameters ${arg1}, ${arg2}, in second listener`);
+});
+// Third listener
+myEmitter.on('event', function thirdListener(...args) {
+  let parameters = '';
+  args.forEach((arg) => {
+    parameters += `${arg}, `;
+  }
+  );
```

Comment:
Please move this closing bracket to the line above. It's otherwise a bit confusing.

It could also be simplified by using `join`:

```js
  const parameters = args.join(', ');
```